src/components/Pivot/Pivot.scss

```diff
@@ -123,6 +123,14 @@
 
     &.is-selected {
       font-weight: $ms-font-weight-semilight;
+
+      .ms-Pivot-count {
+        font-weight: $ms-font-weight-regular;
+      }
+    }
+
+    .ms-Pivot-count {
+      @include margin-left(5px);
```

Comment:
Tried the symetrical but for a selected pivot it causes the blue underline to push out on the right past the end of the text. See below:

![symetrical-margin](https://cloud.githubusercontent.com/assets/13757784/17378078/65b5de9e-5971-11e6-93a7-c56bb577ea56.png)